llvm/test/tools/llvm-readobj/ELF/reloc-types-eravm.test

```diff
@@ -0,0 +1,23 @@
+## Test that llvm-readobj shows proper relocation type
```

Comment:
Squash everything into a single commit. _(Just a formally unresolved comment to not accidentally push fix-up separately)_